app/views/playlists.js

```diff
@@ -0,0 +1,41 @@
+import Mn from 'backbone.marionette';
+import PlaylistView from './playlist';
+import application from '../application';
+
+const Playlists = Mn.CompositeView.extend({
+
+    template: require('./templates/playlists'),
+
+    childViewContainer: '#playlist-list',
+
+    childView: PlaylistView,
+
+    ui: {
+        playlists: 'p',
+    },
+
+    events: {
+        'click p': 'changePlaylist',
+    },
+
+    changePlaylist: function(e) {
+        const playlist = $(e.currentTarget);
+        $(".playlists p").removeClass('selected');
+        playlist.addClass('selected');
+        application.headerInfos.set('title', e.currentTarget.innerHTML);
```

Comment:
I suggest the `changePlaylist` should only trigger trigger an event that'll update the appState (in _application.js_). This way, this component has just to lstenTo `appState` to know which is the current playlist, then re-render to set the `selected` class properly.